server/models/OrderItem.ts

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose'
+import uniqueValidator from 'mongoose-unique-validator'
+
+export interface OrderItems extends mongoose.Document {
+    quantity: number
+    orderId: string
+    iceCreamId: string
+}
+
+const OrderItemSchema = new mongoose.Schema<OrderItems>({
+    quantity: {
+        type: Number,
+        required: true,
+    },
+    orderId: {
+        type: String || mongoose.Schema.Types.ObjectId,
+        required: true,
+        ref: 'Order'
```

Comment:
I think we ended up naming our schema 'Orders', so this might need to be pluralized